lib/ord.js

```diff
@@ -31,8 +28,8 @@ const initializeAppConfig = (csn) => {
     const lastUpdate = getRFC3339Date();
     let odmEntity = [];
 
-    // namespace variable value if present in cdsrc.json take it there or else from package.json
-    //if cdsrc.json does not have applicationNamespace, then use just the namespace
+    // use "namespace" from package.json as fallback in case it's not given in cdsrc.json
+    // if cdsrc.json does not have "applicationNamespace", use the "namespace"
```

Comment:
We can remove it. When I read it, it can't help me have deeper understanding. And we have to maintain it when something changed